pkg/backup/item_backupper_test.go

```diff
@@ -309,9 +310,29 @@ func TestResticAnnotationsPersist(t *testing.T) {
 		).(*defaultItemBackupper)
 	)
 
+	var podVolumeBackup1 = &velerov1api.PodVolumeBackup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "volume-1",
+		},
+		Status: velerov1api.PodVolumeBackupStatus{
+			SnapshotID: "snapshot-1",
+		},
+	}
+
+	var podVolumeBackup2 = &velerov1api.PodVolumeBackup{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "volume-2",
+		},
+		Status: velerov1api.PodVolumeBackupStatus{
+			SnapshotID: "snapshot-2",
+		},
+	}
+
+	podVolumeBackups := []*velerov1api.PodVolumeBackup{podVolumeBackup1, podVolumeBackup2}
```

Comment:
doesn't look like there's any need to give `podVolumeBackup1` and `podVolumeBackup2` their own variables - let's just define them inline in the slice definition